includes/admin/class-wc-payments-admin.php

```diff
@@ -54,7 +57,59 @@ public function add_payments_menu() {
 			return;
 		}
 
-		$top_level_link = $stripe_connected ? '/payments/deposits' : '/payments/connect';
+		$navigation_enabled = Loader::is_feature_enabled( 'navigation' );
+		$top_level_link     = $stripe_connected ? '/payments/deposits' : '/payments/connect';
+
+		if ( $navigation_enabled ) {
+			// Register category.
+			Menu::add_plugin_category(
+				[
+					'id'         => 'woocommerce-payments',
+					'title'      => __( 'WooCommerce Payments', 'woocommerce-payments' ),
+					'capability' => 'manage_woocommerce',
+				]
+			);
+
+			if ( ! $stripe_connected ) {
+				Menu::add_plugin_item(
+					[
+						'id'         => 'woocommerce-payments-connect',
+						'title'      => __( 'Connect', 'woocommerce-payments' ),
```

Comment:
In the current implementation, the menu looks like this when WCPay is connected:

```
[$] Payments
  L__ Deposits
  L__ Transactions
  L__ Disputes
  L__ Settings
```

When not connected, the `[$] Payments` menu item is the only one registered, so it acts as an "item" instead of a "category", instead of having a "Payments" category with a single child. Connecting WCPay will result in a full-page reload, so the current behavior could be replicated in the new navigation, right?

Reply:
Okay, so you'd like to update it to a single item as opposed to the category with a "Connect" menu item, right?

>Connecting WCPay will result in a full-page reload, so the current behavior could be replicated in the new navigation, right?

This functionality should remain the same, so items should be updated appropriately.

Reply:
> Okay, so you'd like to update it to a single item as opposed to the category with a "Connect" menu item, right?

That's correct. At that point, there's nothing else to do in WCPay except "Connect", so it's either that, or a category with a single item, which seems pointless.